LifeCycleAssessment_Engine/Compute/ElementEmbodiedCarbon.cs

```diff
@@ -39,16 +40,31 @@ public static partial class Compute
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [PreviousVersion("6.2", "BH.Engine.LifeCycleAssessment.Compute.ElementEmbodiedCarbon(BH.oM.Dimensional.IElementM, System.Collections.Generic.List<BH.oM.Physical.Materials.Material>, System.Boolean)")]
```

Comment:
This method is redundant- you can evaluate for GWP by itself in EvaluateElement. Is this accommodating methods in the Structures tools?

Reply:
Yeah, it is redundant.

I added it in the previous refactor to make it a bit easier to use, without having to populate with various enums.

Will most likely use this for the structures tool if we keep it, and can still see some value in it, given that GWP is the most commonly evaluated metric.

But yeah, it is redundant, but I personally think worth keeping anyway.